contributors/devel/sig-architecture/feature-gates.md

```diff
@@ -9,6 +9,17 @@ Feature gates are intended to cover the development life cycle of a feature - th
 Features generally progress through `Alpha` -> `Beta` -> `GA`. Sometimes we end up deciding that a feature is not going to be supported and we end up marking them as `Deprecated`.
 
 The majority of features will go through all three stages, but occasionally there are features which may skip stages.
+While some exceptions may happen, approvers should use the following guidance:
+- features that involve API changes must progress through all `Alpha`, `Beta`, `GA` stages
+- larger or more complex changes should progress through all `Alpha`, `Beta`, `GA` stages
+- smaller changes that carry non-trivial risk (e.g. due to changing user-facing behavior) might skip
+  `Alpha` and start directly in `Beta` (provided the appropriate `Beta` quality is achieved)
+  but should be off by-default until proven in some production environment
+- smaller changes with low enough risk might skip `Alpha` and start directly in `Beta`
+  (provided the appropriate `Beta` quality is achieved) and can be enabled by-default from the
+  very beginning
+- changes that are explicitly considered as bug fixes are recommended to go directly to `Beta`
+  and should be enabled by-default from the very beginning
```

Comment:
Many bug fixes today are merged without a feature gate at all.

Is the intent here that "bug fixes that have a sufficient level of risk that being able to turn the fix off via a feature gate is justified" should be handled this way?

Reply:
we are using Deprecated state to roll out some bug fixes that may have some risk of breaking , see https://github.com/kubernetes/kubernetes/pull/119789

Reply:
I don't think it actually works in all cases. Let me try to clarify that [yes, I think that bug fixes with sufficient level of risk should actually go with FGs]